vms/platformvm/block/executor/standard_block_test.go

```diff
@@ -364,38 +364,38 @@ func TestBanffStandardBlockUpdateStakers(t *testing.T) {
 	// Staker4:            |------------------------|
 	// Staker5:                                 |--------------------|
 	staker1 := staker{
-		nodeID:        ids.GenerateTestNodeID(),
-		rewardAddress: ids.GenerateTestShortID(),
+		nodeID:        ids.BuildTestNodeID([]byte{0xf1}),
```

Comment:
Aligned this test with proposal block one above. After all they test the very same things, just on different block types